src/algorithms/calorimetry/CalorimeterHitDigi.h

```diff
@@ -48,7 +52,7 @@ class CalorimeterHitDigi {
 
     // digitization settings
     unsigned int             m_capADC;
-    double                   m_capTime = 1000.;
+    double                   m_capTime = 1000. * unit::ns;
```

Comment:
Is whatever uses this aware that units are now included?

Reply:
This was originally introduced in https://github.com/eic/EICrecon/pull/591/files#r1163225587 by @FriederikeBock where we didn't understand why dd4hep units didn't work.

We are not really using this cutoff. I don't think that the timing information is propagated correctly, and we don't simulate any signal shaping, so the exact value is a moot. I don't change the number, just declare the unit value.